app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java

```diff
@@ -61,22 +55,20 @@ public void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().setDisplayUseLogoEnabled(true);
         getSupportActionBar().setTitle(R.string.about);
 
-
         mIndicator.setOnPageChangeListener(this);
 
-        mViewPagerAdapter = new AboutPagerAdapter(this, getSupportFragmentManager());
-        mViewPager.setAdapter(mViewPagerAdapter);
+        mViewPager.setAdapter(new AboutPagerAdapter(this, getSupportFragmentManager()));
         mIndicator.setViewPager(mViewPager);
     }
 
     protected String getTrackedViewName() {
-        return "About/"+getResources().getStringArray(R.array.about_tabs)[getCurrentPage()];
+        return "About/" + getResources().getStringArray(R.array.about_tabs)[getCurrentPage()];

```

Comment:
True, but I didn't want to change that as part of this PR. The change is only due to code reformatting. I will eventually refactor that code also. :)